src/pages/whats-happening/v11-release/index.mdx

```diff
@@ -0,0 +1,195 @@
+
+## Read more in Medium
+
+<Row>
+<Column colLg={4} colMd={4} noGutterMdLeft>
+<ArticleCard
+    title="Carbon’s 2021 release: April update"
+    author="Josh Black"
+    date="April 22, 2021"
+    href="https://medium.com/carbondesign/carbons-2021-release-april-update-9d23242b3dea"
+    >
+
+![Carbon's 2021 release: April update](images/carbon_2021-april-update.png)
+
+</ArticleCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterMdLeft>
+<ArticleCard
+    title="What’s coming to Carbon in 2021"
+    author="Josh Black"
+    date="November 12, 2020"
+    href="https://medium.com/carbondesign/whats-coming-to-carbon-in-2021-39a4c7d1762a"
+    >
+
+![Carbon 2021 release](images/carbon_2021.png)
+
+</ArticleCard>
+</Column>
+</Row>
+
+## Release FAQs
+
+### What are the benefits of Carbon v11 for me as a designer?
+
+- Token names have been updated for easier application. Names are more logical
+  and easier to apply without always having to refer to a guide.
+
+* Nothing visually has changed in any of the components, and so no redesign work
+  is required.
+
+### What are the benefits of Carbon v11 for me as a developer?
+
+- CSS grid provides easier implementation of layouts, support for sub-grid, and
+  other features that come along with this new technology.
+
+* CSS custom properties for theming (light mode / dark mode).
+
+- A 90% reduction in compile times for most projects with the update to Sass
+  architecture.
+
+* Quality of life updates such as:
+  - Consistent component API across the board.
+  - Improvements to Sass structure for getting started and productive use
+    experiences.
+
+- New components:
+  - Cover a wider variety of use-cases in your product with some of our new
+    components.
+  - Easily build our more accessible experiences with our accessibility
+    component primitives.
+
+### What are the benefits to my product's customers?
+
+- Light/dark mode support.
+
+* Faster load times due to smaller bundle sizes.
+
+### Do I need to update right away? If not, when will we need to update?
+
+- Carbon v11 includes functionality that may be a motivator for migration, but
+  teams can migrate to v11 when they have the bandwidth.
+
+* Teams that are using v10 today can continue to stay on v10 and everything that
+  is implemented will continue to work.
+
+### What will the Carbon team be supporting for v10?
+
+- No new functionality will be introduced in v10 after the v11 release.
+
+* We will continue to address high impact bugs that come up for the v10 release
+  after the v11 release.
+
+- We will continue to accept any contributions that look to address issues in
+  the v10 release if we are unable to get to them in time for your product
+  roadmap.
+
+### What is the rule for supporting deprecated things?
+
+- Things that were deprecated from v9 to v10 will be removed in v11.
+
+* Things that are deprecated in v10 will remain in v11 and will be removed in
+  v12.
```

Comment:
I'm wondering if we should state our intent with timing of v12. We don't know when that would happen, and like v11 we want to delay breaking changes as long as it makes sense.

In the past, our rule was "major versions at minimum six months apart". But having gone over 2 years between v10 and v11, it seems we're set up for a slower frequency of major versions.

Maybe something like:

>The timeline of v12 has not been set, but it's the team's intent for major versions to be at minimum 12 months apart.

Thoughts @alisonjoseph @joshblack?

Reply:
Added. Let me know if tweaks needed.

Reply:
Yes, I don't see us doing another major release anytime before next year, and even then, likely longer.